lighthouse-core/lib/image-helpers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @param {string} src
+ * @return {boolean}
+ */
+function isSVG(src) {
```

Comment:
nit: rename to `isSVGUrl` ? And should this be in url-shim.js, since this isn't really specific to images but URLs?

Reply:
I was thinking we still put this on `mimeType` until v9 and extract `/\.(png|jpeg|jpg|svg|webp|gif|avif)(\?.*)?$/.match(url)[1]` so no usage changes are required